templates/spin_sidebar.hbs

```diff
@@ -107,7 +107,7 @@
                     <li><a {{#if (active_content request.spin-path-info "/spin/url-shortener" )}} class="active" {{/if}}
                             href="{{site.info.base_url}}/spin/url-shortener">Building a URL Shortener With Spin</a></li>
                     <li><a {{#if (active_content request.spin-path-info "/spin/spin-oci" )}} class="active" {{/if}}
-                            href="{{site.info.base_url}}/spin/spin-oci">Hosting Spin Apps in OCI Registries</a></li>
+                            href="{{site.info.base_url}}/spin/spin-oci">Spin Apps in Registries</a></li>
```

Comment:
For me this reads better with some sort of verb indicating the activity but subjective I know!

```suggestion
                            href="{{site.info.base_url}}/spin/spin-oci">Hosting Spin Apps in Registries</a></li>
```